packages/fuels/src/cli/templates/fuels.config.hbs

```diff
@@ -0,0 +1,167 @@
+import type {
+  CommandEvent,
+  // DeployOptions,
+  FuelsConfig,
+} from 'fuels';
+
+import { createConfig } from 'fuels';
+
+export default createConfig({
+
+  // ----------------------------------- Inputs
+
+  /**
+   * Instead of informing `contracts`, `scripts` and `predicates`
+   * individually, you can also use workspaces (recommended)
+   * @param workspace - Path to Sway/Forc Workspace
+   */
+  {{#if (isDefined workspace)}}
+  workspace: '{{workspace}}',
+  {{else}}
+  // workspace: './backend',
+  {{/if}}
+
+
+  /**
+   * This property should not be used alongside `workspace`
+   * @param contracts - Paths to Sway Contracts
+   */
+  {{#if (isDefined contracts)}}
+  contracts: [
+    {{#each contracts}}
+      '{{this}}',
+    {{/each}}
+  ],
+  {{else}}
+  // contracts: [],
+  {{/if}}
+
+  /**
+   * This property should not be used alongside `workspace`
+   * @param predicates - Paths to Sway Predicates
+   */
+  {{#if (isDefined predicates)}}
+  predicates: [
+    {{#each predicates}}
+      '{{this}}',
+    {{/each}}
+  ],
+  {{else}}
+  // predicates: [],
+  {{/if}}
+
+  /**
+   * This property should not be used alongside `workspace`
+   * @param scripts - Path to Sway Scripts
+   */
+  {{#if (isDefined scripts)}}
+  scripts: [
+    {{#each scripts}}
+      '{{this}}',
+    {{/each}}
+  ],
+  {{else}}
+  // scripts: [],
+  {{/if}}
+
+  /**
+   * @param output - Where to generate Typescript definitions
+   */
+  output: '{{output}}',
+
+
+  // ----------------------------------- Node & Deployment settings
+
+  /**
+   * @param privateKey - Your wallet private key
+   */
+  // Should ideally come from env — `process.env.MY_PRIVATE_KEY`
+  privateKey: '0xa449b1ffee0e2205fa924c6740cc48b3b473aa28587df6dab12abc245d1f5298',
```

Comment:
I think this privateKey is here because it is one of the default accounts on the chainConfig, making it better for users testing it.

I think we should change the default accounts on the chainConfig, to use these ones, if is not a big push for this PR: https://github.com/FuelLabs/fuels-portal/blob/main/docker/fuel-core/accounts.json
Also, add these accounts somewhere in the logs when we start a fuel-core using it...

If is maybe just do on the next PR.

Reply:
[@camsjams This property became optional and will be validated when [necessary](https://github.com/FuelLabs/fuels-ts/blob/ee60da654305e5d70faafca147f0c89f6e83512a/packages/fuels/src/cli/commands/deploy/createWallet.ts#L12).

@luizstacio Yes, I copied it from other tests. It's the same as the `consensusKey`.

----

 - https://github.com/FuelLabs/fuels-ts/issues/1373